scripts/commits

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python3
+
+import sys
```

Comment:
smuggled this in, clears up history confusions and parses merge trains specially